std/php/Global.hx

```diff
@@ -931,6 +931,21 @@ extern class Global {
 	**/
 	static function header(string:String, replace:Bool = true, ?http_response_code:Int):Void;
 
+	/**
+		@see http://php.net/manual/en/function.header-remove.php
+	**/
+	static function header_remove(?name:String):Void;
+
+	/**
+		@see http://php.net/manual/en/function.headers-list.php
+	**/
+	static function headers_list():Array<String>;
+
+	/**
+		@see http://php.net/manual/en/function.headers-sent.php
+	**/
+	static function headers_sent(?file:String, ?line:Int):Bool;
```

Comment:
arguments should be `?file:Ref<String>, ?line:Ref<Int>`

Reply:
I was wondering how to specify arguments passed by reference. Thanks for the correction.